pytests/test_paid_parental_leave.py

```diff
@@ -0,0 +1,228 @@
+from . import Reasoner
+
+
+class TestPaidParentalLeave(Reasoner):
+    key = 'isPaidParentalLeave'
+
+
+class TestPaidParentalLeaveDefault(TestPaidParentalLeave):
+    body = {}
+
+    def test_default(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestPaidParentalLeaveEligible(TestPaidParentalLeave):
+    body = {
+        "applicant": {
+            "isStoppingWorkToCareForChild": True,
+            "gaveBirthToThisChild": True,
+            "meetsPaidParentalLeaveEmployedRequirements": True
+        },
+        "child": {
+            "Age": 0
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_permitted)
+
+
+class TestPaidParentalLeaveAdoptedEligible(TestPaidParentalLeave):
+    body = {
+        "applicant": {
+            "isStoppingWorkToCareForChild": True,
+            "meetsPaidParentalLeaveEmployedRequirements": True
+        },
+        "child": {
+            "Age": 1,
+            "AdoptedByApplicant": True
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_permitted)
+
+
+class TestPaidParentalLeaveAdoptedNotStoppingWork(TestPaidParentalLeave):
+    body = {
+        "applicant": {
+            "isStoppingWorkToCareForChild": False,
+            # "meetsPaidParentalLeaveEmployedRequirements": False
+        },
+        "child": {
+            "Age": 1,
+            "AdoptedByApplicant": True
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestPaidParentalLeaveAdoptedNotWorkingTenHours(TestPaidParentalLeave):
+    body = {
+        "applicant": {
+            "isStoppingWorkToCareForChild": True,
+            "meetsPaidParentalLeaveEmployedRequirements": False
+        },
+        "child": {
+            "Age": 1,
+            "AdoptedByApplicant": True
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestPaidParentalNotEmployed(TestPaidParentalLeave):
+    body = {
+        "applicant": {
+            "isStoppingWorkToCareForChild": True,
+            "gaveBirthToThisChild": True,
+            "meetsPaidParentalLeaveEmployedRequirements": False
+        },
+        "child": {
+            "Age": 0
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_forbidden)
+
+
+class TestPaidParentalTransferredToSpouse(TestPaidParentalLeave):
+    body = {
+        "applicant": {
+            "isStoppingWorkToCareForChild": True,
+            "gaveBirthToThisChild": False,
+            "meetsPaidParentalLeaveEmployedRequirements": True
+        },
+        "spouse": {
+            "transferringEntitlementToApplicant": True,
+            "gaveBirthToThisChild": True,
+            "meetsPaidParentalLeaveRequirements": True
+        },
+        "child": {
+            "Age": 0
+        }
+    }
+
+    def test_reasoning(self):
+        self.assertTrue(self.is_permitted)
+
+# TODO needs clarification
```

Comment:
@Br3nda Is this PR done? noticed this TODO

Reply:
Yes, I've got an email thread with MBIE to get clarification on the PPL rules.
It's not clarified yet. For now the spouse of the mother rules on RaaP are in but then the "catch all" is is catching them permitting PPL.